radixdlt/foo-hack
# Post production / refactoring / optimisations.

- Code splitting (rolled out into components).
- Added SCSS.
- Bug Fixes.
- UX Optimisations (such as redirects on transaction success).
- Tested 👍

Further optimisations, NFT's and Auction COMING UP! 😁

Todo's below:

- Re-enable frontend move validation.
- Player turn mech.
- Match draw functionality and alt GIF implementation.
- Ensure that piece movement is consistent (don't update state until component response).
- Add the rewards / NFT auction mech.
- Customise board and pieces.
- Detect if wallet / account is not present.
